Google.Api.Generator/RoslynUtils/RoslynExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.CodeAnalysis;
+using System.Linq;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Google.Api.Generator.RoslynUtils
+{
+    internal static class RoslynExtensions
+    {
+        public static T WithTrailingSpace<T>(this T node) where T : SyntaxNode => node.WithTrailingTrivia(Space);
```

Comment:
I've found the ordering and grouping of these to be odd. It's also a bit weird that we only need a generic WithLeadingCrlf, and only need a non-generic WithLeadingSpace, but I guess that's just the way it happens.

Reply:
I don't see any particular ordering/grouping being particularly better than others unfortunately.
The methods here are the ones that are required so far.
Some of the "missing" methods will be added later once more generation code is added.